tests/models/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copy the models to the build directory.
+
+file(GLOB files RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} caffe2Models/*.pbtxt)
```

Comment:
how do you plan to keep more test models under caffe2Models folder?

Reply:
In general, I would like to 1) group ops; 2) try to share the weights file. 
After the conv op testing is done, I would like to create small networks starting from a conv layer for testing (i.e based on some real models but reduce the layers). Therefore, the weights file can be shared. When an new op is ready for testing, if it is proper, it can be added into the existing models as a layer. Otherwise, a new model file should be created.